SWSH/Program.cs

```diff
@@ -261,8 +261,9 @@ private static void __connect()
             {
                 if (File.ReadAllLines(_mainDirectory + nickname + ".swsh")[0] == "-password")
                 {
-                    Console.Write($"Password for {nickname}: ");
-                    ccinfo = __CreateConnectionInfoPassword(nickname, __getCommand());
+                    ReadLine.PasswordMode = true;
```

Comment:
This commit will fail the tests because its missing an assembly reference. I can't add it because I don't have VS.